api/Query/RescueQuery.js

```diff
@@ -53,7 +54,7 @@ class RescueQuery extends Query {
         where: rats,
         model: Rat,
         as: 'rats',
-        required: false,
+        required: !isEmpty(rats),
```

Comment:
This behaviour can be acheived through the use of an array destructuring assignment, please use that instead of bringing in underscore.